.github/workflows/android_deploy_production.yml

```diff
@@ -34,9 +34,9 @@ jobs:
 
       - name: Set up .env
         env:
-          ENV_PRODUCTION: $
+          ENV_PRODUCTION: ${{ secrets.ENV_PRODUCTION }}
         run: |
-          echo $ENV_PRODUCTION > .env
+          echo $ENV_PRODUCTION | base64 --decode > .env
```

Comment:
@AVI5HEK, You keep data with encoding `base64`, right? Why do you need to `decode` here 🤔

Reply:
@Wadeewee The file is saved in the secrets as a base64 string. So we need to decode it back to the `.env` file before building the app.

Reply:
Thanks for the confirmation @AVI5HEK 🙏